lib/viz/chart/chart.ts

```diff
@@ -10,8 +10,13 @@ import Loader = require('../loader');
 import Formatters = require('../formatters');
 import Dom = require('../dom');
 import ResultHandling = require('../result-handling');
+import c3 = require('c3');
 import Classes = require('../css-classes');
 
+if(!c3) { 
+    throw "c3 has not been loaded."; 
```

Comment:
Maybe we could say "c3 is not loaded - you must include the Connect fork of c3 for visualization to function correctly".

On that - how can we make sure they're not just including any old c3?